services/common/src/components/permits/PermitConditionForm.tsx

```diff
@@ -322,7 +322,7 @@ const PermitConditionForm: FC<PermitConditionFormProps> = ({
                 </Col>
                 <Col className="condition-column" {...editableProps}>
                     <Col className="condition-editor">
-                        <VariableConditionMenu inputRef={conditionInputRef} isManagementView={Boolean(standardConditionType)} conditionForm={editingFormName}/>
+                        {stepEditDisabled && <VariableConditionMenu inputRef={conditionInputRef} isManagementView={Boolean(standardConditionType)} conditionForm={editingFormName}/>}
```

Comment:
Yeah, I kind of would prefer to rename this to something like 'showVariableConditionMenu'. Even if it means creating a second identical boolean by a different name.  This feels a bit confusing here.